regression/tests/lms/test_dashboard.py

```diff
@@ -24,9 +25,9 @@ def test_resume_course(self):
         """
         Verifies that we can successfully resume the course
         """
-        course_name = 'course-v1:ArbiRaees+AR-1000+fall'
-        course_page = CourseInfoPageExtended(self.browser, course_name)
-        courseware_page = CoursewarePage(self.browser, course_name)
+        course_id = 'course-v1:ArbiRaees+AR-1000+fall'
```

Comment:
Since you've declared this as a variable in the StudioHome page, can you import it, or put it in a shared location?